meinberlin/apps/projects/templatetags/meinberlin_project_tags.py

```diff
@@ -53,36 +52,70 @@ def get_sorted_modules(context):
     )
 
 
-@register.simple_tag
-def get_num_entries(module):
-    """Count all user-generated items."""
-    item_count = (
-        Idea.objects.filter(module=module).count()
-        + MapIdea.objects.filter(module=module).count()
-        + budget_proposal.objects.filter(module=module).count()
-        + kiezkasse_proposal.objects.filter(module=module).count()
-        + Vote.objects.filter(choice__question__poll__module=module).count()
-        + LiveQuestion.objects.filter(module=module).count()
-        + Like.objects.filter(question__module=module).count()
-    )
-    comment_filter = (
-        Q(idea__module=module)
-        | Q(mapidea__module=module)
-        | Q(budget_proposal__module=module)
-        | Q(kiezkasse_proposal__module=module)
-        | Q(topic__module=module)
-        | Q(maptopic__module=module)
-        | Q(paragraph__chapter__module=module)
-        | Q(chapter__module=module)
-        | Q(poll__module=module)
-    )
-    comment_count = (
-        Comment.objects.filter(comment_filter)
-        .annotate(child_comment_count=Count("child_comments__pk", distinct=True))
-        .aggregate(comment_count=Count("pk") + Sum("child_comment_count"))[
-            "comment_count"
-        ]
+@register.inclusion_tag("meinberlin_projects/includes/module-tile/module_insights.html")
```

Comment:
what's the benefit of adding the template here rather than in the parent template? I feel is more transparent and consitent to have the templates explicitly included within templates.

Reply:
You mean `{% include ... %}` vs `inclusion_tag`? I found it was a perfect case for an `inclusion_tag` because it always comes with its own `template` assigned, so it's more modular; in case you ever want to display this elsewhere, you could just call the tag.

Let me know however, if I should make the switch to `{% include %}`